x-pack/legacy/plugins/spaces/public/views/management/edit_space/customize_space/customize_space_avatar.tsx

```diff
@@ -20,6 +33,7 @@ interface Props {
 interface State {
   initialsHasFocus: boolean;
   pendingInitials?: string | null;
+  imageUrl?: string | null;
```

Comment:
Do we still need `imageUrl` as part of this component state? I thought we were going to try to use `this.props.space.imageUrl` instead, since the two values appear to be kept in sync

Reply:
I can't remove imageUrl from "State". ts gives me an error:

Type '{ initialsHasFocus: false; imageUrl: string | undefined; }' is not assignable to type 'Readonly<State>'.
  Object literal may only specify known properties, and 'imageUrl' does not exist in type 'Readonly<State>'.ts(2322)

Reply:
Did you also remove all references to `this.state.imageUrl`, and `this.setState({ imageUrl: '...' })` within the component?

Reply:
Ehm... no :) Now it's done.